### PR TITLE
Add property years_avail to ReadEMEP

### PR DESCRIPTION
**Pull request by ejgal**
Add property to match changes in colocation routines (#173)

### CODE REVIEW

**Comment by ejgal** [comment]
Should give correct result, since all files in data_dir should be from the same year.
@jgliss, do you have a better idea for this?

File: pyaerocom/io/read_emep.py [hunk 1]
```diff
@@ -121,6 +122,21 @@ def ts_types(self):
...
+        Years available in dataset
+        """
+        try:
+            paths = self._get_paths()
+            data = xr.open_dataset(paths[0])
+            years = data.time.dt.year.values
+            years = list(np.unique(years))
+            return sorted(years)
+        except Exception as e:
+            return []
```
Comment (jgliss) [inline pyaerocom/io/read_emep.py:137]: Seems okay and is probably not very critical for processing, is it?

**Review by jgliss — APPROVED** [review]
I think that's okay! For AeroCom reading it is more important to have `years_available` in the reader, as files are single year, and for computation of auxiliary variables the correct file matches have to be found. In case of EMEP, all variables are stored in a single netcdf file, and thereby share the same time dimension, correct?

**Comment by ejgal** [comment]
Yes, that's the idea. This was done to work with _infer_start_stop from colocation_auto, so should only be accessed if start is not provided.

**Pull request merged**